packages/devextreme/js/__internal/ui/chat/messagebubble.ts

```diff
@@ -23,11 +31,33 @@ class MessageBubble extends Widget<Properties> {
 
     super._initMarkup();
 
-    this._updateText();
+    this._updateContent();
   }
 
-  _updateText(): void {
-    const { text = '' } = this.option();
+  _updateContent(): void {
+    const {
+      text = '', template = null, isLast, author,
+    } = this.option();
+
+    if (template) {
+      $(this.element()).empty();
+
+      const messageTemplate = this._getTemplateByOption('template');
+
+      const templateData = {
+        component: this,
```

Comment:
[incorrect behavior]
Component should point to the Chat, not private Bubble.
I suggest you create smth like `messageTemplateData` option/method on a Chat level to pass component and author here

Reply:
0e11dd012baad48fce62eeab905c60abc39aa9d5